manifests/sysctl.pp

```diff
@@ -233,25 +235,28 @@
         sysctl { 'kernel.exec-shield': value => $kernel__exec_shield }
       }
 
-      unless $ipv6.is_a(Undef) {
+      if $ipv6.is_a(Undef) {
```

Comment:
It just wasn't correct before. I found this as I was running through the acceptance testing.